src/main/kotlin/demo/exceptions/ControllerExceptionHandler.kt

```diff
@@ -0,0 +1,14 @@
+package demo.exceptions
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ControllerAdvice
+import org.springframework.web.bind.annotation.ExceptionHandler
+
+@ControllerAdvice
+class ControllerExceptionHandler {
+    @ExceptionHandler(Exception::class)
+    fun handleException(): ResponseEntity<Any> {
+        return ResponseEntity("Unknown error occurred", HttpStatus.BAD_REQUEST)
```

Comment:
Jako, że tylko my tego używamy, to może zostać, ale Unknown error jest raczej z naszej strony, czyli powinno być coś w stylu 5xx

Reply:
Może faktycznie niefortunny komentarz, miałem na myśli złapanie tym błędem np próbę wysłania nieprawidłowego uuid